src/coffee/io/reader.coffee

```diff
@@ -6,18 +6,26 @@ iconv = require 'iconv-lite'
 fs = Promise.promisifyAll require('fs')
 Excel = require 'exceljs'
 
+debugLog = _.noop
+
 class Reader
 
   constructor: (@options = {}) ->
+    if @options.debug
+      debugLog = console.log
```

Comment:
Just a quick one I just thought.

Is it possible that we use NodeJS default debug module for this

https://nodejs.org/api/debugger.html

It's inbuilt and a bit better than _console.log_